src/main/java/world/jnc/invsync/util/serializer/module/mod/SolCarrotSyncModule.java

```diff
@@ -0,0 +1,95 @@
+package world.jnc.invsync.util.serializer.module.mod;
+
+import com.cazsius.solcarrot.capability.FoodCapability;
+import com.cazsius.solcarrot.capability.FoodInstance;
+import com.cazsius.solcarrot.handler.HandlerCapability;
+import com.cazsius.solcarrot.handler.MaxHealthHandler;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+import lombok.experimental.UtilityClass;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.Item;
+import org.spongepowered.api.data.DataContainer;
+import org.spongepowered.api.data.DataQuery;
+import org.spongepowered.api.data.DataView;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.item.inventory.ItemStack;
+import org.spongepowered.common.item.inventory.util.ItemStackUtil;
+import world.jnc.invsync.util.serializer.NativeInventorySerializer;
+
+public class SolCarrotSyncModule extends BaseModSyncModule {
+  @Override
+  public String getModId() {
+    return "solcarrot";
+  }
+
+  @Override
+  public DataView serialize(Player player, DataView container) {
+    return Helper.serialize(player, container);
+  }
+
+  @Override
+  public void deserialize(Player player, DataView container) {
+    Helper.deserialize(player, container);
+  }
+
+  @UtilityClass
+  private static class Helper {
+    private static final DataQuery STACK = DataQuery.of("stack");
+    private static final DataQuery META = DataQuery.of("meta");
+
+    private static DataView serialize(Player player, DataView container) {
+      FoodCapability foodCapability =
+          NativeInventorySerializer.getNativePlayer(player)
+              .getCapability(FoodCapability.FOOD_CAPABILITY, null);
+      List<DataView> serializedFoods = new LinkedList<>();
+      if (foodCapability != null) {
```

Comment:
Blank lines haven't killed anyone yet ;)

Have a little look at the rest of the code to see what spacing style I usually use.

And lastly if you haven't done that yet (though it looks like you did) run `./gradlew spotlessApply` to have correct formatting.

Reply:
I added some new lines. Let me know if it still doesn't match your style.

Yes, I ran `gradlew spotlessApply`. That's a nice tool. I didn't know about it until it printed warnings.